python/ray/tune/examples/mnist_pytorch.py

```diff
@@ -103,8 +104,11 @@ def train_mnist(config):
     while True:
         train(model, optimizer, train_loader, device)
         acc = test(model, test_loader, device)
-        # Set this to run Tune.
-        session.report({"mean_accuracy": acc})
+        checkpoint = None
+        if should_checkpoint:
+            checkpoint = Checkpoint.from_dict({"model_state_dict": model.state_dict()})
```

Comment:
should we use TorchCheckpoint.from_state_dict here?

Reply:
Yup, I think it makes sense to use `TorchCheckpoint` and then `TorchPredictor` in the model loading example later. Will change.